Unable to run repo using Apple Silicon (probably?)

## Description

I'm facing trouble to run **assets** container. It's stopping on 60% and not doing something else:

<img width="1512" alt="Снимок экрана 2024-05-20 в 17 32 52" src="https://github.com/mozilla/bedrock/assets/18268746/51a5f183-ec22-4b37-9178-ea05eaa149fe">

It's also consumes full 1 CPU core, but nothing happens even after an hour of payload:

<img width="1033" alt="Снимок экрана 2024-05-20 в 17 42 12" src="https://github.com/mozilla/bedrock/assets/18268746/6518275e-d656-490f-aab5-a3ddf7ded9ae">

### Steps to reproduce

**Requires:**
- Macbook M1-3 (probably?)
- Actual repo, master branch

**What to do:**
```bash
cd bedrock
make clean install
```

### Expected result

2 running containers with container 'assets' and compiled webpack modules in it

### Actual result

```log
assets-1  | <s> [webpack.Progress] 59% building 364/408 entries 1269/1304 dependencies 701/788 modules
```

### Environment

- Safari/Google Chrome/Firefox browsers are tested
- macOS Sonoma 14.4.1 (23E224)
- M3 Max processor

### Notes

I'm really sorry if I doesn't met issue design/category requirements, it's my first impulse to contribute to open source

@alexgibson No, I didn't

I've tried this right now, and it's working, thank you!

But I got some errors for now after entire process was done:

```
WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/

ERROR in ./media/js/glean/elements.es6.js 8:0-52
Module not found: Error: Can't resolve '../libs/glean/element.js' in '/app/media/js/glean'
resolve '../libs/glean/element.js' in '/app/media/js/glean'
  using description file: /app/package.json (relative path: ./media/js/glean)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /app/package.json (relative path: ./media/js/libs/glean/element.js)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/element.js doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/element.js.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/element.js.json doesn't exist
      .wasm
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/element.js.wasm doesn't exist
      as directory
        /app/media/js/libs/glean/element.js doesn't exist
 @ ./media/js/glean/init.es6.js 10:0-59 61:6-17

ERROR in ./media/js/glean/page.es6.js 8:0-46
Module not found: Error: Can't resolve '../libs/glean/page.js' in '/app/media/js/glean'
resolve '../libs/glean/page.js' in '/app/media/js/glean'
  using description file: /app/package.json (relative path: ./media/js/glean)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /app/package.json (relative path: ./media/js/libs/glean/page.js)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/page.js doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/page.js.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/page.js.json doesn't exist
      .wasm
        Field 'browser' doesn't contain a valid alias configuration
        /app/media/js/libs/glean/page.js.wasm doesn't exist
      as directory
        /app/media/js/libs/glean/page.js doesn't exist
 @ ./media/js/glean/init.es6.js 9:0-78 33:2-25 58:6-16

webpack 5.91.0 compiled with 2 errors and 3 warnings in 132662 ms
```

--- a/package.json
+++ b/package.json
@@ -51,7 +51,6 @@
     "webpack-dev-server": "^5.0.4"
   },
   "scripts": {
-    "prestart": "npm run glean",
     "start": "concurrently --kill-others \"python manage.py runserver 0.0.0.0:8080\" \"npm run watch\"",
     "in-pocket-mode": "SITE_MODE=Pocket npm start",
     "lint-js": "npx eslint .",
@@ -63,7 +62,7 @@
     "static": "webpack --config webpack.static.config.js --mode=production --bail",
     "prebuild": "npm run glean && npm run static",
     "build": "webpack --mode=production --bail --progress",
-    "prewatch": "npm run static",
+    "prewatch": "npm run glean && npm run static",
     "watch": "webpack serve --mode=development --progress",
     "prettier": "prettier --write .",
     "prettier-check": "prettier --check .",

